src/components/api/apiSlice.js

```diff
@@ -0,0 +1,73 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const apiSlice = createApi({
+  reducerPath: 'Bikes',
+  baseQuery: fetchBaseQuery({
+    baseUrl: 'https://bike-rental-api.onrender.com/api/v1',
```

Comment:
- [ ] This production/cloud onrender URL did not work for me.  It didn't let me access it and I wasn't able to create a user:

![image](https://github.com/soulemanou-software/car-rental-frontend/assets/57421823/03b875f0-d518-4d6b-8459-beca9a1ef1ae)


However, the local server seems to be working perfectly.  I changed the `baseUrl` to 'http://127.0.0.1:3000/api/v1' and the project worked without issue:

![image](https://github.com/soulemanou-software/car-rental-frontend/assets/57421823/98d3ee09-ee2f-4044-a118-e3220718e6e4)


If `onrender` is giving you trouble in this matter, please, kindly use the local rails server.  It works fine. 💪

Reply:
THqnks